scripts/src/main/resources/scripts/command/cobigen

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+# $@: optional maven arguments
+function doSetup() {
+  # We need a JDK
+  doDevonCommand java -q setup
+  # We need Maven for installing plug-ins
+  doDevonCommand mvn -q setup
+  if [ -n "${1}" ] || [ ! -d "${COBIGEN_CLI_HOME}" ]
+  then
+    local software_version="LATEST"
+    local download_url="https://repository.sonatype.org/service/local/artifact/maven/redirect?r=central-proxy&g=com.devonfw.cobigen&a=templates-devon4j&v=${software_version}"
```

Comment:
This is the download URL I need to change. Right now I'm just downloading CobiGen templates and not the CLI.